doc/user_guide/customization.rst

```diff
@@ -290,3 +290,88 @@ You can remove the legend entirely by submitting a null value.
       y='petalLength',
       color=alt.Color('species', legend=None),
   )
+
+Adjusting the width of Bar Marks
+--------------------------------
+The width of the bars in a bar plot are controlled through the `size` property in the `mark_bar()`:
+
+.. altair-plot::
+
+  import altair as alt
+  import pandas as pd
+
+  data = pd.DataFrame({'name': ['a', 'b'], 'value': [4, 10]})
+
+  alt.Chart(data).mark_bar(size=10).encode(
+      x='name:O',
+      y='value:Q'
+  )
+
+But since `mark_bar(size=10)` only controls the width of the bars, it might become possible that the width of the chart is not adjusted accordingly:
+
+.. altair-plot::
+
+  import altair as alt
+  import pandas as pd
+
+  data = pd.DataFrame({'name': ['a', 'b'], 'value': [4, 10]})
```

Comment:
This data should still be available to this plot directive since it is defined in the previous one... no need to re-define it here.

Same comment applies in several places below.